### ISSUE TITLE
KML/GPX support broken on Windows

### ISSUE THREAD

**Issue by wilhelmberg** [mapbox/mapbox-studio-classic#1473]
Since 0.2.8 it is not possible to open KMLs anymore (verified with 0.2.7 and 0.3.1).

They fail with

```
Error: OGR source missing extent
```

I tracked it down to the `mapnik-omnivore` function `getExtent` where

``` js
  var tranformation = new gdal.CoordinateTransformation(
    this.gdalLayer.srs,
    gdal.SpatialReference.fromEPSG(4326)
  );
```

fails with `Unable to open EPSG support file gcs.csv. Try setting GDAL_DATA.`:

![image](https://cloud.githubusercontent.com/assets/4549888/10304776/7b829b6a-6c1c-11e5-8436-9805950d4e3e.png)

`GDAL_DATA` is set correctly and the directory, that `GDAL_DATA` points to contains `gcs.csv`.

Using `mapnik-omnivore` alone (not in the context of Mapbox Studio Classic) `digest()` works as expected and returns the correct extent.

Maybe `postinstall` step `npm dedupe` messes something up.

Investigating ...

/cc @amyleew

**Comment by snowpong** [comment]
Is there a workaround for existing users of 0.3.3 for Windows? It fails with the same error.

**Comment by wilhelmberg** [comment]
@snowpong not at the moment, but the [underlying issue](https://github.com/naturalatlas/node-gdal/pull/134) has been resolved and next Studio Classic version (https://github.com/mapbox/mapbox-studio-classic/issues/1491) should contain the fix.

In the meantime you could use `QGIS` or `ogr2ogr` to convert the KML to a shapefile and work with that.

```
ogr2ogr -f "ESRI Shapefile" out.shp YOUR.kml -t_srs EPSG:3857
```

**Comment by mannylopez** [comment]
Also happens on Mac when trying to upload [run.gpx](https://www.mapbox.com/help/data/run.gpx). I also get `Error: OGR source missing extent`.

I was able to upload the same file via the [uploads page](https://www.mapbox.com/studio/data) so the data is good.

**Comment by springmeyer** [comment]
@mannylopez - yes, this is fixed in mb-pages branch, but needs to get released still for users.

**Comment by synergywp** [comment]
Anything new with this?  Thanks.

**Comment by springmeyer** [comment]
This should be fixed in v0.3.4 now that we have upgraded to latest node-gdal (v0.8.0) with this fix: https://github.com/naturalatlas/node-gdal/pull/134. Keeping open until either me or @mapsam confirms via manual testing.

**Comment by springmeyer** [comment]
confirmed via local testing that KML is now working in v0.3.4 and GPX is working except for GPX with `<time>` element. That fix will need to wait on https://github.com/mapbox/mapbox-studio-classic/issues/1543.
